src/rows.c

```diff
@@ -51,22 +51,31 @@ size_t compute_row_size(int actual_num_fields,
 }
 
 
-PyObject *call_converter_function(PyObject *func, char32_t *token)
+PyObject *call_converter_function(PyObject *func, char32_t *token, bool byte_converters)
 {
     Py_ssize_t tokenlen = 0;
     while (token[tokenlen]) {
         ++tokenlen;
```

Comment:
Isn't this `strlen`?

Reply:
You mean the while-loop? I think so? That's not anything I changed though so if you want to clean it up here (or in a different PR) sounds good to me :+1:

Reply:
don't worry about it, no point to fix here probably...